be/src/agent/heartbeat_server.cpp

```diff
@@ -81,6 +81,7 @@ void HeartbeatServer::heartbeat(THeartbeatResult& heartbeat_result,
         heartbeat_result.backend_info.__set_be_node_role(config::be_node_role);
         // If be is gracefully stop, then k_doris_exist is set to true
         heartbeat_result.backend_info.__set_is_shutdown(doris::k_doris_exit);
+        heartbeat_result.backend_info.__set_be_process_uuid(_be_epoch);
```

Comment:
why not use start time?

Reply:
its it same, be start time is assigned to _be_epoch either.